meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -312,6 +312,7 @@ MACHINE_FIRMWARE_mx8mm = "linux-firmware-imx-sdma-imx7d"
 WKS_FILE_mx8mm_sota = "sdimage-imx8-sota.wks.in"
 ## iMX8MM EVK
 MACHINE_FEATURES_remove_imx8mmevk = "qca9377 qca6174"
+IMXBOOT_TARGETS = "flash_spl"
```

Comment:
This should probably be specific to mx8mm, so IMXBOOT_TARGETS_mx8mm or similar.

Reply:
ah yeah, sorry, forgot about the prefix